etcd-client/Cargo.toml

```diff
@@ -30,6 +30,8 @@ priority-queue = "1.0.5"
 protobuf = "2.16.2"
 smol = "1.2.4"
 thiserror = "1.0"
+crossbeam-queue = "0.3.1"
```

Comment:
Why not use the latest version of this library?

Reply:
Let me check it.